fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/resources/Permissions.java

```diff
@@ -27,19 +27,19 @@
 
 /**
  * Representation of authorization configuration for a resource. This object is immutable, which
- * makes it challenging when working with Jackson's {{ObjectMapper}} and Spring's
- * {{\@ConfigurationProperties}}. The {@link Builder} is a helper class for the latter use case.
+ * makes it challenging when working with Jackson's {@code ObjectMapper} and Spring's
+ * {@code @ConfigurationProperties}. The {@link Builder} is a helper class for the latter use case.
  */
 @ToString
 @EqualsAndHashCode
 public class Permissions {
 
-  public static Permissions EMPTY = new Permissions.Builder().build();
+  public static final Permissions EMPTY = Builder.fromMap(Collections.emptyMap());
```

Comment:
I think `new Builder().build()` is a little more straightforward personally, but feel free to ignore...

Reply:
The reason I moved away from new Builder().build() was because build() returns this instance if it builds an empty permission object

Reply:
Oh yeah, duh, good point thanks :grin: